### PR TITLE
possibly speed up implementation of isIdentifierStart/Part functions

### PR DESCRIPTION
**Pull request by michaelficarra**
/cc @ikarienator

### CODE REVIEW

**Comment by michaelficarra** [comment]
JSPerf confirms it is faster: http://jsperf.com/test-identifier-part

File: lib/code.js [hunk 1]
```diff
@@ -87,34 +87,37 @@
...
     function isIdentifierPartES6(ch) {
-        return (ch >= 97 && ch <= 122) ||     // a..z
-            (ch >= 65 && ch <= 90) ||         // A..Z
-            (ch >= 48 && ch <= 57) ||         // 0..9
-            (ch === 36) || (ch === 95) ||     // $ (dollar) and _ (underscore)
-            ((ch >= 0x80) && ES6Regex.NonAsciiIdentifierPart.test(fromCodePoint(ch)));
+        return ch < 0x80 ? IDENTIFIER_PART[ch] : ES6Regex.NonAsciiIdentifierPart.test(fromCodePoint(ch));
     }
 
     module.exports = {
```
Comment (mathiasbynens) [inline lib/code.js]: Let’s use `0x`-style integer literals whenever referencing Unicode code points.

**Comment by Constellation** [comment]
Really nice :)

File: lib/code.js (hunk 1 above)
Comment (michaelficarra) [inline lib/code.js]: Done.

**Pull request merged**